### DISCUSSION TITLE
Navigation Between TabView using code Behind

### DISCUSSION THREAD

**Discussion by faizkhan005** [enisn/UraniumUI#757] [Q&A] [answered]
Hi,

I Have a requirement where I need to navigate between tabs based on a button press.
tried googling but could not find any solution, also tried setting selectedTab but could not get the tabitem instance 

is there a way I can navigate between tabs ?

**Comment by enisn** [comment, marked as answer]
You can do something like that:

```csharp
tabView.SelectedTab = tabView.Items[2];
```

**Reply by faizkhan005** [reply]
It worked 
Thank you so much
